src/TourManager/TourManager.Repository.EfCore/Entities/TourClient.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace TourManager.Repository.EfCore.Entities
+{
+    public class TourClient
+    {
+        public long Id { get; set; }
+
```

Comment:
should our ids be typeof long? @HAghajanyan-ST

Reply:
Yes Tom jan, I think it's ok, but you are right it can be `int` as well .